_includes/description.html

```diff
@@ -1,11 +1,19 @@
 <h2>Meet the Pod</h2>
-<div class="card-container">
-    {% for item in site.data.fellows %}
-    <div class="card">
-        <div class="info">
-            <p class="title"> {{ item.name }}</p>
-            <p class="location"> {{ item.location }}</p>
+<div class="row people d-flex justify-content-center">
+  {% for item in site.data.fellows %}
+  <div class="col-md-4 col-lg-3 item">
+    <div class="box" style="background-image: url(./assets/img/{{ item.img }})">
+      <div class="cover">
+        <h3 class="name">{{ item.name }}</h3>
+        <p class="title">{{ item.location }}</p>
+        <div class="social">
+          <a href={{ item.linkedin }}><i class="fa fa-linkedin-in"></i></a>
```

Comment:
I think it might be best to conditionally render the icons only if they have respective links to be clicked. Otherwise, users will be clicking the icons and thinking the site is broken 😱

Reply:
Noted, will make the changes.